apps/ledger-live-mobile/src/families/cosmos/accountActions.tsx

```diff
@@ -27,7 +37,11 @@ const getActions = ({
               ? ScreenName.CosmosDelegationValidator
               : ScreenName.CosmosDelegationStarted,
         },
-      ],
+      ];
+  return [
+    {
+      id: "stake",
+      navigationParams: navigationParams as unknown as NavigationParamsType,
```

Comment:
This isn't ideal, what type error are you getting without this casting?

Reply:
I agree but I do not find a better solution.
Not type:
```
Type '(NavigatorName | { screen: ScreenName; params: { account: CosmosAccount; parentAccount: Account | undefined; }; })[] | (NavigatorName | { ...; })[]' is not assignable to type 'readonly [name: string, options: object] | undefined
```

as NavigationParamsType
```
Conversion of type '(NavigatorName | { screen: ScreenName; params: { account: CosmosAccount; parentAccount: Account | undefined; }; })[] | (NavigatorName | { ...; })[]' to type 'NavigationParamsType' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```

as const (as it is defined in other part of the app)
```
Type '(NavigatorName | { screen: ScreenName; params: { account: CosmosAccount; parentAccount: Account | undefined; }; })[] | (NavigatorName | { ...; })[]' is not assignable to type 'readonly [name: string, options: object] | undefined'
```